build/README.md

```diff
@@ -531,5 +531,8 @@ Changes in v3.1.0
 * Added a new `writer` field to identification_service_areas and subscriptions table for the purposes of automatic Garbage Collection 
 * Read the database version and cache it in memory. 
 
+Changes in v3.1.1
+* Add index for subscriptions table by end time.
+
 The `writer` column was added to facilitate automatic Garbage Collection, recording the instance that added the record to Database.
-The presence `writer` column in the Database will not cause failure in basic functionality of the DSS, as long as you keep the version of the Server ahead of the Database version. When running Server v3.1.0 with Database v3.0.0, the `writer` column will simply be ignored and that record will not be automatically Garbage collected. You will need to manually remove expired records either with the existing Delete API ro manual SQL commands.
\ No newline at end of file
+The presence `writer` column in the Database will not cause failure in basic functionality of the DSS, as long as you keep the version of the Server ahead of the Database version. When running Server v3.1.x with Database v3.0.0, the `writer` column will simply be ignored and that record will not be automatically Garbage collected. You will need to manually remove expired records either with the existing Delete API ro manual SQL commands.
```

Comment:
How about just, "Only since commit ??? on Aug ??, 2020 will the DSS enable automatic garbage collection of records by tracking which DSS instance is responsible for garbage collection of the record.  Expired records added with a DSS deployment running code earlier than this must be manually removed."